packages/contracts-bedrock/src/L2/L2ToL2CrossDomainMessenger.sol

```diff
@@ -162,14 +178,17 @@ contract L2ToL2CrossDomainMessenger is ISemver, TransientReentrancyAware {
         CrossL2Inbox(Predeploys.CROSS_L2_INBOX).validateMessage(_id, keccak256(_sentMessage));
 
         // Decode the payload
-        (uint256 destination, address target, uint256 nonce, address sender, bytes memory message) =
+        (uint256 destination, address target, uint256 nonce, address sender, bytes memory message, address entrypoint) =
             _decodeSentMessagePayload(_sentMessage);
 
         // Assert invariants on the message
         if (destination != block.chainid) revert MessageDestinationNotRelayChain();
         if (target == Predeploys.CROSS_L2_INBOX) revert MessageTargetCrossL2Inbox();
         if (target == Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER) revert MessageTargetL2ToL2CrossDomainMessenger();
 
+        // Ensure an entrypoint was not specified or that the entrypoint is the msg.sender
+        if (entrypoint != address(0) && entrypoint != msg.sender) revert InvalidEntrypoint();
```

Comment:
Just a NIT here, lmk wdyt:
The first comment says you will check the invariants and you declare them all together.

Then you add a blank line and put the entrypoints check. Maybe it can be understood as that it is another extra check but not part of the invariants? I'd remove that blank line:

```suggestion
        // Assert invariants on the message
        if (destination != block.chainid) revert MessageDestinationNotRelayChain();
        if (target == Predeploys.CROSS_L2_INBOX) revert MessageTargetCrossL2Inbox();
        if (target == Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER) revert MessageTargetL2ToL2CrossDomainMessenger();
        // Ensure an entrypoint was not specified or that the entrypoint is the msg.sender
        if (entrypoint != address(0) && entrypoint != msg.sender) revert InvalidEntrypoint();
```

Reply:
100% Will change it!